packages/wrangler/src/config/validation-helpers.ts

```diff
@@ -236,7 +236,7 @@ export const isString: ValidatorFn = (diagnostics, field, value) => {
  */
 export const isValidName: ValidatorFn = (diagnostics, field, value) => {
 	if (
-		(typeof value === "string" && /^$|^[a-z0-9_ ][a-z0-9-_ ]*$/.test(value)) ||
+		(typeof value === "string" && /^$|^[a-z0-9_][a-z0-9-_]*$/.test(value)) ||
```

Comment:
Change looks good. Maybe a good time to add some unit tests here?

Reply:
I've adapted the existing test to ensure it catches this case